railties/lib/rails/generators/rails/app/templates/dockerignore.tt

```diff
@@ -6,6 +6,10 @@
 # Ignore bundler config.
 /.bundle
 
+# Ignore all environment files (except templates).
+/.env*
+!/.env*.erb
```

Comment:
There's also another entry for `.env` files 5 lines below

Reply:
Ah, my bad, will clean.